sansio_lsp_client/client.py

```diff
@@ -116,7 +126,7 @@ class ClientState(enum.Enum):
         },
         "workDoneProgress": True,
     },
-    "workspace": {
+"workspace": {
```

Comment:
I'm not sure why the CI checks aren't running. Black should complain about this.